koin-projects/examples/multimodule-ktor/module-b/src/main/kotlin/org/koin/samples/multi_module/module_b/moduleB.kt

```diff
@@ -0,0 +1,28 @@
+package org.koin.samples.multi_module.module_b
+
+import io.ktor.application.Application
+import io.ktor.application.call
+import io.ktor.response.respondText
+import io.ktor.routing.get
+import io.ktor.routing.routing
+import org.koin.dsl.bind
+import org.koin.dsl.module
+import org.koin.experimental.builder.single
+import org.koin.ktor.ext.inject
+import org.koin.ktor.ext.koin
+import org.koin.samples.multi_module.common.IService
+
+fun Application.moduleB() {
+    koin {
+        modules(module {
+            single<ModuleBService>() bind IService::class
+        })
+    }
```

Comment:
same remark here about the Koin DSL exposure

Reply:
Good suggestion with koinModules - I will add this to DSL